src/js/pages/jobs/JobsTable.js

```diff
@@ -42,17 +42,94 @@ class JobsTable extends React.Component {
         prop: 'name',
         render: this.renderHeadline,
         sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
+        sortFunction: function (prop, direction) {
+          let score = 1;
+
+          if (direction === 'desc') {
+            score = -1;
+          }
+
+          return function (a, b) {
+            // Hoist group trees to the top
+            if (a.isGroup && !b.isGroup) {
+              return score * -1;
+            } else if (b.isGroup && !a.isGroup) {
+              return score;
+            }
+
+            return a.name.localeCompare(b.name);
+          };
+        },
+        heading
+      }, {
+        className,
+        headerClassName: className,
+        prop: 'status',
+        render: function (prop, row) {
```

Comment:
What do you think about splitting these into their own methods seeing thats what we do elsewhere?